src/js/overlay.js

```diff
@@ -89,6 +89,11 @@ const triggerClickHandler = function(id, ev) {
  * @param {Boolean} opts.fullscreen - If set to true, the overlay will display full screen. This overlay disables scroll on the underlying document and is dismissible with the back button.
 */
 const Overlay = function(id, opts) {
+	if (overlays[id]) {
+		// @todo Throw error in the next breaking.
```

Comment:
Could you open an issue to track this work?

Reply:
The console warning should mention that this will throw in the next major version.